README.md

```diff
@@ -1019,18 +1020,40 @@ The `android` structure contains the following options:
 
 **Example**
 
+```yaml
+- name: maven
+  gradleCliPath: ./gradlew
+  mavenCliPath: scripts/mvnw.cmd
+  mavenSettingsPath: scripts/settings.xml
+  mavenRepoId: ossrh
+  mavenRepoUrl: https://oss.sonatype.org/service/local/staging/deploy/maven2/
+  android:
+    distDirRegex: /^sentry-android-.*$/
+    fileReplaceeRegex: /\d\.\d\.\d(-SNAPSHOT)?/
+    fileReplacerStr: release.aar
+```
+
+### Symbol Collector (`symbol-collector`)
+
+Using the [`symbol-collector`](https://github.com/getsentry/symbol-collector) client, uploads native symbols.
+The client needs to be available in the path.
+
+**Configuration**
+
+| Option           | Description                                                                                  |
+| ---------------- | -------------------------------------------------------------------------------------------- |
+| `serverEndpoint` | **optional** The server endpoint. Defaults to `https://symbol-collector.services.sentry.io`. |
+| `batchType`      | The batch type of the symbols to be uploaded.                                                |
```

Comment:
```suggestion
| `batchType`      | The batch type of the symbols to be uploaded. I.e: `Android`, `macOS`, `iOS`.                                                |
```

Reply:
Added this, but it'd be best to have that documentation somewhere else and point there. Updating Craft's docs because the symbol collector got e.g. a new feature and now supports a new type doesn't seem ideal.